### PR TITLE
Release version 1.0.0

### PR DESCRIPTION
**Pull request by imnutz**
- Update td-ios-sdk and td-android-sdk (v1.0.1)

### CODE REVIEW

**Review by tung-vu-td — APPROVED** [review]
LGTM

**Review by tung-vu-td — APPROVED** [review]

**Review by tung-vu-td — CHANGES_REQUESTED** [review]
This should be a major version as well, to prevent breaking changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.4.0 (2023-07-17)
```
Comment (tung-vu-td) [inline CHANGELOG.md:1]:
```suggestion
## 1.0.0 (2023-07-17)
```

**Review by tung-vu-td — CHANGES_REQUESTED** [review]

**Review by nmpennypacker — APPROVED** [review]
Reviewed for data exfiltrations

**Pull request merged**